actions/send_email.yaml

```diff
@@ -39,5 +39,5 @@
     attachments:
       type: "array"
       items:
-        type: "string"
-      description: "The absolute paths to the files to be included as attachments."
+        type: "Object"
```

Comment:
should `"object"` instead of uppercase `"Object"`, this is why the test failed

Reply:
I don't like simply converting this over, because it would break this action for all users and all workflows that are currently using this action and passing in strings here.

We should be able to use `oneOf` here to preserve backwards compatibility for existing users of this pack. Please do so.

```yaml
    attachments:
      type: array
      description: |
        An array or list of items, either strings or objects/dictionaries, that specifies
        each file to be included as attachments.
      items:
        oneOf:  # this should be camelCase, as defined by JSONSchema
          - type: string
            description: The absolute path to the file to be attached
          - type: object
            description: An object containing file_id and file_path keys
            properties:
              file_id:
                type: string
                description: A unique identifier for the attachment in the mail envelope
              file_path:
                type: string
                description:  The absolute path to the file to be attached
```

Note that `oneOf` should be `camelCase`, as it is defined in JSONSchema, but `file_id` and `file_path` should both be `snake_case`, as they are interpreted by Python in StackStorm.